docs/devices/overview.md

```diff
@@ -22,16 +22,16 @@ Properly configuring devices is an important step to ensuring proper communicati
 
 ![Device Type](/images/devices/device-type.png "Device Type")
 
-The first thing to configure is the type of device. This is not about the actual physical
-type of the device, but more about how you expect the device to connect and report state
-to the Losant Platform. There are three categories ...
+The first thing to configure is the type of device. This is not about the actual physical type of the device, but more about how you expect the device to connect and report state to the Losant Platform. There are four categories ...
 
 * **Standalone**  
-  A standalone device is your normal everyday type of device. It is allowed to connect directly to Losant and report its own state. In most cases, this is the device type you will choose.
+  A Standalone device is your normal everyday type of device. It is allowed to connect directly to Losant and report its own state. In most cases, this is the device type you will choose.
+* **Edge Compute**  
+  An [Edge Compute](/devices/edge-compute/) device is a device that runs the Losant [Edge Agent](/edge-compute/edge-agent-usage/) and can run [edge](/workflows/edge-workflows/) workflows directly on the device itself. They also can behave as Gateway devices, in that they may report state on behalf of a Peripheral device.
```

Comment:
"can run [edge](/workflows/edge-workflows/) workflows "

Nit. the link text should include the word "workflows"

Reply:
👍